addons/website/views/snippets/s_floating_blocks.xml

```diff
@@ -0,0 +1,174 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_floating_blocks" name="Floating Cards">
+    <section class="s_floating_blocks pt32 pb32 o_cc o_cc2">
+        <div class="container">
+            <div class="d-flex flex-column gap-5">
+                <span class="s_floating_blocks_wrapper s_floating_blocks_wrapper_shadow d-contents rounded-3">
+
+                    <section data-name="Card"
+                        class="o_block s_col_no_resize position-sticky d-flex overflow-hidden py-5 o_cc o_cc5 oe_img_bg o_bg_img_center o_background_video"
+                        data-bg-video-src="https://player.vimeo.com/video/329369394?autoplay=1&amp;muted=1&amp;autopause=0&amp;controls=0&amp;loop=1"
+                        style="background-image:url(https://i.vimeocdn.com/video/774173730-b7a179626fb9429551c395a423571c56515633a3cdbf18261054f281a022b19d-d?mw=80&amp;q=85)">
```

Comment:
Do you absolutely need to use a video by default? It will be annoying as we cannot add external references in tours.

Reply:
I've replaced the "placeholder image" (prior to video starts) with a better static one.
I'd keep the video tho, it's nice to showcase that we provide these features :)